tests/spread/core22/components/task.yaml

```diff
@@ -0,0 +1,45 @@
+summary: Build a snap with components
+
+prepare: |
+  snap refresh snapd --edge
+
+restore: |
+  snapcraft clean --destructive-mode
+  rm -f ./*.snap ./*.comp
+  snap refresh snapd --stable
+
+execute: |
+  # Craft Parts is not partitions aware during organize in managed mode
```

Comment:
https://github.com/canonical/craft-parts/pull/689 makes this test pass without needing `--destructive-mode`

Reply:
might as well have @cmatsuoka release that tomorrow and I can just ensure we update the test here